perception/sub8_perception/CMakeLists.txt

```diff
@@ -133,6 +133,10 @@ install(DIRECTORY include/${PROJECT_NAME}/
 #     "src/sub8_perception/*.cc"
 # )
 
+file(GLOB START_GATE_SOURCES
+      "src/sub8_perception/start_gate.cc"
```

Comment:
If we all wrote in assembly, there would be no compilation errors and we would get better performance. I'm for this.